src/app/zap-templates/zcl/data-model/chip/tls-client-management-cluster.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2025 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/tls/TLSClientManagement.adoc
+Parameters: in-progress 
+Git: 1.4-819-g9894ad1b9
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="General"/>
+  <enum name="TLSEndpointStatusEnum" type="enum8">
+    <cluster code="0x0802"/>
+    <item name="Provisioned" value="0x00"/>
+    <item name="InUse" value="0x01"/>
+  </enum>
+
+  <struct name="TLSEndpointStruct" apiMaturity="provisional">
+    <cluster code="0x0802"/>
+    <item fieldId="0" name="EndpointID" type="int16u"/>
+    <item fieldId="1" name="Hostname" type="octet_string"/>
+    <item fieldId="2" name="Port" type="int16u"/>
+    <item fieldId="3" name="CAID" type="int16u"/>
+    <item fieldId="4" name="CCDID" type="int16u" isNullable="true" optional="true"/>
+    <item fieldId="5" name="Status" type="TLSEndpointStatusEnum" min="0x00" max="0x01"/>
+  </struct>
+
+  <cluster apiMaturity="provisional">
+    <domain name="Cameras"/>
```

Comment:
The TLS domain should not be Cameras I believe.

Reply:
Good call, had my mind so much on cameras didn't stop to think that this can be used outside of it